### PR TITLE
Fix: Enable properly starting Rock Racketeer

### PR DESCRIPTION
**Pull request by davismj**

### CODE REVIEW

**Comment by davismj** [comment]
please hold off on pulling this in

**Comment by davismj** [comment]
I had to rescript significant portions of this quest. It could use a quick code review.

File: scripts/zones/Palborough_Mines/npcs/Mythril_Seam.lua [hunk 1]
```diff
@@ -89,11 +75,18 @@ end;
 function onEventFinish(player,csid,option)
     -- printf("CSID: %u",csid);
     -- printf("RESULT: %u",option);
-    
-    if (csid == 0x002b) then
-        player:tradeComplete();
+
+    player:tradeComplete();
```
Comment (takhlaq) [inline scripts/zones/Palborough_Mines/npcs/Mythril_Seam.lua]: you'll want to move this inside the appropriate cutscene check

Comment (davismj) [inline scripts/zones/Palborough_Mines/npcs/Mythril_Seam.lua]: good call

**Pull request merged**